credmark/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.8.22'
```

Comment:
This __init__.py file cannot be here. credmark folder is a namespace.

Reply:
The version is related to the cmf folder anyhow.

Reply:
You could have a try of https://pypi.org/project/versioneer/
https://github.com/python-versioneer/python-versioneer